package.json

```diff
@@ -43,6 +43,7 @@
     "bchaddrjs": "0.5.2",
     "bignumber.js": "9.0.2",
     "bip32": "3.0.1",
+    "bip86": "0.0.1",
```

Comment:
Latest version is actualy 0.0.2: https://www.npmjs.com/package/bip86

Reply:
yes, most packages need to be updated https://github.com/LedgerHQ/xpub-scan/actions/runs/3227498359/jobs/5282389858#step:4:54
I want to create separate PR for packages update - to know that if smth will fail it will fail because of versions update